### PR TITLE
5 upsert team data

### LINKED ISSUE #5: Upsert Team data
Modify pipeline to extract and load data for teams.

### PR DESCRIPTION
**Pull request by steinarkarlsson**
Teams data is now upserted to mongoDB

### CODE REVIEW

File: etl/config.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+username = 'steinar'
```
Comment (James-Leslie) [inline etl/config.py]: This shouldn't be in the repo

File: etl/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.config
```
Comment (James-Leslie) [inline etl/.gitignore]: I think this needs to be `config.py`

**Review by James-Leslie — CHANGES_REQUESTED** [review]

**Review by James-Leslie — APPROVED** [review]

File: etl/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+config.py
```
Comment (steinarkarlsson) [inline etl/.gitignore]: Amended config to config.py

File: etl/config.py (hunk 1 above)
Comment (steinarkarlsson) [reply etl/config.py]: I will untrack this file in git once the .gitignore is updated

**Pull request merged**